src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/TargetFrameworkProvider.cs

```diff
@@ -3,71 +3,32 @@
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
 using System.Linq;
-using Microsoft.VisualStudio.ComponentModelHost;
-using Microsoft.VisualStudio.Shell;
 using NuGet.VisualStudio;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.CrossTarget
 {
     [Export(typeof(ITargetFrameworkProvider))]
     [AppliesTo(ProjectCapability.DependenciesTree)]
-    internal class TargetFrameworkProvider : OnceInitializedOnceDisposed, ITargetFrameworkProvider
+    internal sealed class TargetFrameworkProvider : ITargetFrameworkProvider
```

Comment:
This class used to have protected members on it but had no derived class.  I changed those members to private and saw no case where we need to derive from this class.  I can remove it :)